app/containers/Button/index.tsx

```diff
@@ -64,7 +64,7 @@ const Button: React.FC<IButtonProps> = ({
 		style
 	];
 
-	const textStyle = [styles.text, { color: isDisabled ? colors.fontDisabled : resolvedTextColor, fontSize }, styleText];
+	const textStyle = [styles.text, { color: isDisabled ? colors.fontWhite : resolvedTextColor, fontSize }, styleText];
```

Comment:
I might be wrong, but it looks like you're bypassing semantic to make UI fit figma.
That's not the idea.
Figma says this should be `Button/Font/on-primary-disabled`.
If our colors are outdated, you should update them.

Reply:
Yeah, you are right, I was confused about this too, but decided since this change is only meant for the disabled button text color, so might be okay to just change the color where used.

I'll update the `fontDisabled` color to white.